main.go

```diff
@@ -36,9 +37,12 @@ func main() {
 	r.Use(cors.Handler(options))
 
 	// Define routes
-	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprint(w, WELCOME_MESSAGE)
-	})
+	// r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+	// 	fmt.Fprint(w, WELCOME_MESSAGE)
+	// })
```

Comment:
I think we can leave this here (don't comment/delete it). It's good to have an endpoint that is always active as a "health check" – ensure that the server is actually up and not broken or some sort.